blockchain-introduction.md

```diff
@@ -0,0 +1,35 @@
+# Blockchain introduction
+
+Cosmos already has a [great intro on blockchains](https://cosmos.network/intro) so you can start there
+This page is more about how Agoric sees and uses blockchains
+
+## A blockchain is like a machine
+
+> a blockchain is a machine which state is made of consensus, not of silicon
+
+(source?)
+
+![A Blockchain, validators each with its ledger copy](./images/blockchain.svg)
+
+A blockchain can be described as a **digital ledger** maintained by a set of **validators**. Each square above is one of these **validators** (a physical machine) and each has its own copy of the **ledger**
+
+The blockchain network acts like a single virtual machine
+
+
+## Communicating with this machine
+
+### Solo node
+
+[Cosmic SwingSet](https://github.com/Agoric/cosmic-swingset) allows to build a **solo node**. This solo node can communicate with the rest of the blockchain network and add transactions to it.
```

Comment:
on line 16, `blockchain network` referred to a single chain. Here it seems to mean the ecosystem consisting of many chains.  Either usage would be reasonable, but they're inconsistent with each other.

Reply:
> Here it seems to mean the ecosystem consisting of many chains

It's not my intention
In this document i'm only talking about single blockchain

Maybe that the term "blockchain network" is confusing. Would it be better it i said "validator network"?